### DISCUSSION TITLE
onKeyStroke documentation page makes typing A,W,S.D in search box not work

### DISCUSSION THREAD

**Discussion by timtoo** [vueuse/vueuse#1951] [General]
Presumably because the A,W,S,D keys are being used for the [move-the-ball demo](https://vueuse.org/core/onkeystroke/#demo), if you try to use the page's search box you can't type the letters A,W,S,D into the input. This is very confusing, until one realizes what is happening there!

Not sure how to make the demo ignore the search box input. Maybe some `useActiveElement` trick, like the `useMagicKeys` documentation describes?

**Comment by curtgrimes** [comment]
The same thing happens for me. I submitted a fix for this: https://github.com/vueuse/vueuse/pull/2050
